common/usermode.c

```diff
@@ -105,10 +117,31 @@ void init_usermode(percpu_t *percpu) {
     vmap_user_4k(usermode_helpers, virt_to_mfn(usermode_helpers), L1_PROT);
 
     init_syscall();
+    init_sysenter();
```

Comment:
Should we test if the CPU actually supports SEP or simply assume we're running on a non-ancient CPU?

Reply:
We have this ancient issue #13, which is supposed to add proper enumerations of CPU flags and features and then use it accordingly. I would handle this there (when finally someone decides to implement it ;-))